Sources/OpenAPIAsyncHTTPClient/AsyncHTTPClientTransport.swift

```diff
@@ -32,7 +32,13 @@ import protocol Foundation.LocalizedError
 ///
 /// Instantiate the transport:
 ///
-///     let transport = AsyncHTTPClientTransport()
+///     let httpClient = HTTPClient()
+///     defer {
+///         try! httpClient.syncShutdown()
```

Comment:
we prefer the async `shutdown()` method e.g. https://github.com/swift-server/async-http-client/blob/4e74fefd1cc4e747e7989cbe6f125553d3f526e0/Examples/GetJSON/GetJSON.swift#L50

Reply:
Oh I'd use that too, but in this example I need to shut down from a defer, and I don't think I can call async functions from there.